extension/options.js

```diff
@@ -79,9 +79,32 @@ const templateOption = {
   processComment(commentData, commentElem) {}
 };
 
+const hideUsersOption = {
+  key: 'hideUsers',
+  default: '',
+  createCachedSet(userString) {
+    this.cachedSet = new Set(userString.split(',').map((e) => e.trim()).filter((x) => x));
+  },
+  onValueChange(newValue) {
+    this.createCachedSet(newValue);
+    reprocessComments(this.key);
+  },
+  onStart(currentValue) {
+    this.createCachedSet(currentValue);
+  },
+  processComment(commentData, commentElem) {
+    if (this.cachedSet.has(commentData.name)) {
+      commentElem.classList.add('hidden');
+    } else {
+      commentElem.classList.remove('hidden');
+    }
```

Comment:
This can be expressed a bit more compactly/idiomatically as:

```
commentElem.classList.toggle('hidden', this.cachedSet.has(commentData.name));
 ```